src/stream/src/from_proto/hash_agg.rs

```diff
@@ -45,6 +47,7 @@ impl<S: StateStore> HashKeyDispatcher for HashAggExecutorDispatcher<S> {
             args.input,
             args.agg_calls,
             args.pk_indices,
+            args.actor_id,
```

Comment:
May introduce an executor context to store actor_id as well as other common info shared by all executors.

Reply:
Count on @xxchan 's `Context`. 😄